### PR TITLE
Patch 5

### PR DESCRIPTION
**Pull request by mvibhamenon**
This Pull Request resolves Issue #24 

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

**Comment by mvibhamenon** [comment]
done..

File: participants/vibha.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello world
```
Comment (MJ10) [inline participants/vibha.txt]: Please move this file to the `Participants` directory and add your info as described in the issue. Also rename the file to your Github username

File: Quicksort.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#include<stdio.h>
```
Comment (MJ10) [inline Quicksort.c]: Please move this file to the `QuickSort` directory and rename it to your github username

File: Mergesort.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
```
Comment (MJ10) [inline Mergesort.c]: Please move this file to the `Merge_Sort` directory and rename it to your github username

**Comment by MJ10** [comment]
@mvibhamenon Any updates or should we close this and mark it as spam?

**Pull request closed without merging**